app/controllers/api/v1/visualization/explore_controller.rb

```diff
@@ -116,6 +116,13 @@ def bam_file_info
           }
         end
 
+        def study_user_info
+          render json: {
+            annotations: AnnotationVizService.available_annotations(@study, cluster: nil, current_user: current_api_user),
+            canEdit: @study.can_edit?(current_api_user)
```

Comment:
Non-blocking: we should use the Ruby convention for formatting (snake case) and call this `can_edit`.

Reply:
IIRC, we decided not to use snake case on endpoints explicitly designed for javascript consumption.  See, e.g. the `get_study_explore_info` method.  

I'd be ok revisiting that decision, since it tends to be easier to just assume "anything we get from the API is snake-cased" rather than have to know "models get returned snake_cased, custom APIs are camel".  For this PR, though, I think it's good practice to just match the other explore_controller methods

Reply:
added to engineering strategy rolling agenda